src/main/java/Duke.java

```diff
@@ -22,7 +22,7 @@ public static void exit() {
     }
 
     public static void main(String[] args) {
-        ArrayList<String> userTexts = new ArrayList<String>();
+        ArrayList<Task> tasks = new ArrayList<Task>();
 
         greeting();
         Scanner sc = new Scanner(System.in);
```

Comment:
Maybe there is a better way to name the Scanner variable? Personally I would prefer "scan" over "sc".